packages/textlint-tester/test/ts-setup.js

```diff
@@ -0,0 +1,3 @@
+require("ts-node").register({
+    project: __dirname
+});
```

Comment:
Do we need `typeCheck: true` here? (issue #448 and pull request #449, also ref. #451)

Reply:
Yes. Fix it

Reply:
Thanks!